libfuse/mounter.go

```diff
@@ -20,20 +20,51 @@ type mounter struct {
 	c       *fuse.Conn
 }
 
+const darwinInstallerPath = "/Applications/Keybase.app/Contents/Resources/" +
```

Comment:
Having the app executable path hardcoded here makes me very nervous, because 1) the desktop team can change it any time, and not update this, and 2) it's kind of a separation of powers abuse, right?  kbfsfuse shouldn't really know about the app.  It would seem better to me to either pass in the command for remounting as a CLI flag or an env var, or to just re-build the logic in Go, in `mounter_osx.go`.  It can't be too complicated, right?

Reply:
> or to just re-build the logic in Go

I think it involves talking to the helper, which is probably why core does a similar thing too: https://github.com/keybase/client/blob/45834bd25f739a98ff404e61d181d7e393955721/go/install/install_darwin.go#L803

But having a flag sounds good too. I'll try to update.

Reply:
Ah.  The core side looks like it's just in the installer though, which is what I'd expect.  I doubt the service itself knows about app-level stuff.

Reply:
It get's used lines below: https://github.com/keybase/client/blob/45834bd25f739a98ff404e61d181d7e393955721/go/install/install_darwin.go#L807

Which is the same thing we use here. Perhaps we can move that into a separate package and vendor it here?

Reply:
Both those links are in the installer, which I don't think it's used by the background Keybase service.  Is it?

Reply:
Oh I see what you mean now. I think it is what service uses to call the obj-c written installer to talk to the helper. For example: https://github.com/keybase/client/blob/45834bd25f739a98ff404e61d181d7e393955721/go/keybase/main.go#L188

Reply:
Oh, hrm.  Yuck.  Thanks for the link.